examples/loftq_finetuning/quantize_save_load.py

```diff
@@ -180,6 +180,22 @@ def quantize_and_save():
 
     print_model(base_model, "base_model")
 
+    # convert safetensor to bin
```

Comment:
Hmm, why is this needed? Also, would `lora_model.save_pretrained(lora_model_dir, safe_serialization=False)` also work?